Fixes errors with new autotools

| automake: error: using '$(srcdir)' in TESTS is currently broken: '$(srcdir)/auto-test'
| autoreconf: automake failed with exit status: 1
| ERROR: autoreconf execution failed.

Upstream-Status: Not Submitted
Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: gnome-vfs-2.24.4/test/Makefile.am
===================================================================
--- gnome-vfs-2.24.4.orig/test/Makefile.am	2013-10-17 16:43:36.677017764 -0700
+++ gnome-vfs-2.24.4/test/Makefile.am	2013-10-17 16:48:47.369008566 -0700
@@ -64,7 +64,7 @@
 	test-async-cancel \
 	test-escape       \
 	test-uri       	  \
-	$(srcdir)/auto-test	
+	auto-test
 
 libraries =						\
 	$(top_builddir)/libgnomevfs/libgnomevfs-2.la	\
